docs/src/bootc-install.md

```diff
@@ -222,6 +225,39 @@ a different sandboxing tool (e.g. [bubblewrap](https://github.com/containers/bub
 This argument is mainly useful for 3rd-party tooling for building disk images from bootable
 containers (e.g. based on [osbuild](https://github.com/osbuild/osbuild)).   
 
+
+## Finding and configuring the physical root filesystem
+
+On a bootc system, the "physical root" is different from
+the "logical root" of the booted container. For more on
+that, see [filesystem](filesystem.md). This section
+is about how the physical root filesystem is discovered.
+
+Systems using systemd will often default to using
+[systemd-fstab-generator](https://www.freedesktop.org/software/systemd/man/latest/systemd-fstab-generator.html)
+and/or [systemd-gpt-auto-generator](https://www.freedesktop.org/software/systemd/man/latest/systemd-gpt-auto-generator.html#).
+Support for the latter though for the root filesystem is conditional on EFI and a bootloader implementing the bootloader interface.
+
+Outside of the discoverable partition model, a common baseline default for installers is to set `root=UUID=`
+(and optionally `rootflags=`) kernel arguments as machine specific state.
+When using `install to-filesystem`, you should provide these as explicit
+kernel arguments.
```

Comment:
Can you give an example?  This stuff is outside my comfort zone, so having some example would increase my understanding.

Reply:
This section of the document is targeted at people doing generic installers, disk image image generation tools and custom installs in general.

There's multiple ways to set kernel arguments that are described in the docs already right?